main.js

```diff
@@ -0,0 +1,32 @@
+const navItems = document.querySelectorAll("nav ul li");
+console.log(navItems);
+
+for(let el of navItems){
+    if(el.textContent.includes("All Posts")){
+        el.remove();
+    }
+}
+
+const spanEls = document.querySelectorAll("p span");
+console.log(spanEls);
+
+for(let el of spanEls){
+    if(el.textContent.includes("Read Time")){
+        el.remove();
+    }
+}
+
+const h3tags = document.querySelectorAll(".posts article");
+console.log(h3tags);
+
+for(let el of h3tags){
+    if(el.textContent.includes("You -- yes you!")){
+        el.remove();
+    }
+}
```

Comment:
Interesting way to do this one, nice job! Here's another way in case you're curious:
This way will always grab the last article. 
`const lastEl = h3tags[h3tags.length - 1];` <-- grab the last element of the node list
`lastEl.remove()`                                            <-- and .remove it